tests/Microsoft.Identity.Test.Unit/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -480,27 +480,40 @@ public void AcquireTokenDifferentUserReturnedFromServiceTest()
                             MockHelpers.CreateClientInfo(TestConstants.Uid, TestConstants.Utid + "more"))
                     });
 
-                try
-                {
+                //Ensure Interactive flow does not fail when different home account id is returned
+                AuthenticationResult result2 = app
+                    .AcquireTokenInteractive(TestConstants.s_scope)
+                    .WithAccount(result.Account)
```

Comment:
Need to update the WithAccount XML method to clarify that exception is not thrown for ATI.

https://learn.microsoft.com/en-us/dotnet/api/microsoft.identity.client.acquiretokeninteractiveparameterbuilder.withaccount?view=msal-dotnet-latest#remarks

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/891a8f9eb9c85617e5ae7a0ef69e22f75dace4cd/src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs#L153